pkg/registry/falcon_registry/registry.go

```diff
@@ -117,6 +121,57 @@ func guessLastTag(tags []string, filter func(string) bool) (string, error) {
 func listDockerTags(ctx context.Context, sys *types.SystemContext, imgRef types.ImageReference) ([]string, error) {
 	tags, err := docker.GetRepositoryTags(ctx, sys, imgRef)
 	if err != nil {
+		// Artifactory incorrectly adds the digest to the tag list, so we need to handle this case
+		// by fetching the tags from the registry directly
+		// This is a workaround, and should be removed once Artifactory is fixed
+		if len(tags) == 0 {
+			var jsonRes map[string]interface{}
+			reg, _, _ := strings.Cut(imgRef.StringWithinTransport(), ":")
```

Comment:
We are not checking the return value `found` here.
https://www.codecademy.com/resources/docs/go/strings/cut